### PR TITLE
oak_attestation_verification use prost instead of micro_rpc

### PR DESCRIPTION
**Pull request by ernoc**
#4564

### CODE REVIEW

**Review by tiziano88 — APPROVED** [review]
Thanks

File: oak_attestation_verification/build.rs [hunk 1]
```diff
@@ -14,26 +14,28 @@
 // limitations under the License.
 //
 
+extern crate prost_build;
```
Comment (tiziano88) [inline oak_attestation_verification/build.rs:17]: I don't think this is necessary

Comment (ernoc) [reply oak_attestation_verification/build.rs]: Thanks!

**Review by thmsbinder — APPROVED** [review]
Thanks!

**Pull request merged**